### ISSUE TITLE
Disable Chromium --incognito

### ISSUE THREAD

**Issue by Jamy** [guysoft/FullPageOS#76]
This isn't really an issue, per se, but I have a website application that has an auto login feature that won't work with incognito more.

What file can I edit to make Chromium start without Incognito mode?

Thank you.

**Comment by spencerlandis** [comment]
I'd like this option as well if it persists cookies between boots/sessions.

**Comment by guysoft** [comment]
Yes you can, edit the line ``home/pi/scripts/start_chromium_browser``

The source for comparison is here:
https://github.com/guysoft/FullPageOS/blob/devel/src/filesystem/home/pi/scripts/start_chromium_browser

You can append any flag you want to the chromium call there

**Comment by Jamy** [comment]
I forgot to write back some time ago, but I did notice chromium does not start with incognito mode as default, so it worked well for me.

However, it is very helpful to know the location of the startup script, so thank you for that.
